api/http/management_test.go

```diff
@@ -277,6 +277,119 @@ func TestSetConfiguration(t *testing.T) {
 			}(),
 			Status: http.StatusBadRequest,
 		},
+
```

Comment:
what I am missing in the tests: more than one group in the `RBACHeaderDeploymentsGroups` headers scenario.

Reply:
done